sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -530,11 +530,17 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
     state.cachedAccount = response?.account ?? null;
 
     state.logger.getToken.info(formatSuccess(scopes));
-
-    return {
+    const accessToken: AccessToken = {
       token: response.accessToken,
       expiresOnTimestamp: response.expiresOn.getTime(),
     };
+    if (response.refreshOn) {
```

Comment:
I wonder if we can do something like 

```ts
return {
      token: response.accessToken,
      expiresOnTimestamp: response.expiresOn.getTime(),
      refreshesOnTimestamp: response.refreshOn?.getTime()
}
```

Everywhere? That might be simpler / read better what do you think?

Reply:
Updated to reflect the above